apps/mobile/src/app/(home)/home/home.tsx

```diff
@@ -17,14 +21,22 @@ export function Home() {
   const wallets = useWallets();
   const accounts = useAccounts();
 
+  const query = useBitcoinAccountUtxos();
+
+  console.log({ utxoSumTotal: query });
+
+  const balanceQuery = useBitcoinAccountBalances();
+
+  console.log({ combiledBalance: balanceQuery });
+
   return (
     <PageLayout>
       <AccountsWidget accounts={accounts.list} wallets={wallets.list} />
       <TokensWidget tokens={getMockTokens()} totalBalance={mockTotalBalance} />
-      <CollectiblesWidget
+      {/* <CollectiblesWidget
```

Comment:
Feel free to just delete this `<CollectiblesWidget` and `<TokensWidget` if you want. I'll do it on my next work anyway. 

It seems we don't **have** to do it now on the homepage so lets remove it. I'll get it working on the account page and then we can re-enable once we have more progress on data aggregation. 

That will also get rid of that `source.uri should not be an empty string` message you were seeing